backend/src/main/asciidoc/developer.adoc

```diff
@@ -69,6 +69,13 @@ Contains reference data like an abstraction of symptoms.
 
 include::{moduliths-docs}/module-reference.adoc[]
 
+[[modules.health_departments]]
+=== Health Departments
+
+Contains a service that uses the https://www.rki.de/DE/Content/Infekt/IfSG/Software/Aktueller_Datenbestand.html[data from RKI] to offer the search for health departments by zip code or location.
```

Comment:
the URL is already not reachable and moved and even the ZIP archive is not available any more
https://www.rki.de/DE/Content/Infekt/IfSG/Software/Aktueller_Datenbestand.zip?__blob=publicationFile

Thus essentially means, the data is now "offline". Since we are only talking about relation of ZIP code and health departments, the bundle cached in the project could be used for a lengthy period of time.